src/app/component/matrix/matrix.component.ts

```diff
@@ -160,15 +160,15 @@ export class MatrixComponent implements OnInit {
   rowCtrl = new FormControl('');
   rowCtrlTags = new FormControl('');
   filteredRows: Observable<string[]>;
-  filteredActivity: Observable<string[]>;
+  filteredActivities: Observable<string[]>;
 
   autoCompeteResults: string[] = [];
   autoCompleteActivityResults: string[] = [];
 
   @ViewChild('rowInput') rowInput!: ElementRef<HTMLInputElement>;
   @ViewChild('tagInput') tagInput!: ElementRef<HTMLInputElement>;
 
-  updateActivity(tags: string[]): void {
+  updateActivity(): void {
```

Comment:
can we change the function name to something like `updateActivitesBeingDisplayed` for clarity ?